packages/nextjs/components/Header.tsx

```diff
@@ -54,8 +68,64 @@ export default function Header() {
               Example UI
             </NavLink>
           </li>
+          <li>
+            <NavLink href="/ninja">
+              <BeakerIcon className="h-4 w-4" />
+              Ninja
+            </NavLink>
+          </li>
         </ul>
       </div>
+      <div className="navbar bg-base-100">
```

Comment:
This is a navbar inside a navbar, it should be the same navbar for all (where you show/hide elements depending on the resolution OR two separate sibling navbars)

Also since your aro not hiding it, it makes the menu items to wrap:
![image](https://user-images.githubusercontent.com/2486142/207838079-df5e11bd-f1f8-4c22-8f79-145f7320f2e6.png)

Reply:
Doesn't look good here:
![image](https://user-images.githubusercontent.com/2486142/207840068-d784fa5e-3315-42a1-9db0-6bc89cec90c5.png)

Reply:
On a standard mobile, it looks like this:

![image](https://user-images.githubusercontent.com/2486142/207840332-7d5e276f-65d8-4e7b-8280-52bbe51d2dbd.png)

Reply:
New version: 
![Screenshot from 2022-12-15 16-40-07](https://user-images.githubusercontent.com/21132103/207982524-347a7ab0-b02c-4c91-ae51-f637504df0ca.png)

Doesnt overlap when resized
![Screenshot from 2022-12-15 16-42-16](https://user-images.githubusercontent.com/21132103/207982602-09ee057f-d7f2-42b8-b2cd-ec238448c5ca.png)